tests/test_unaryExpr.py

```diff
@@ -1,11 +1,18 @@
+from os import environ
+
 import pytest
 
 import greenplumpython as gp
 
 
 @pytest.fixture
 def db():
-    db = gp.database(host="localhost", dbname="gpadmin")
+    db = gp.database(
+        host="localhost",
+        dbname="gpadmin",
+        user=environ.get("POSTGRES_USER"),
+        password=environ.get("POSTGRES_PASSWORD"),
+    )
     yield db
     db.close()
```

Comment:
Can we extract this function into a common python file? as it appears in many test files.

Reply:
I think so.

Reply:
will try